airflow/ui/src/router.tsx

```diff
@@ -168,8 +168,7 @@ export const routerConfig = [
   },
 ];
 
-const locationPath = globalThis.window.location.pathname;
-const indexOf = locationPath.indexOf("webapp/");
-const basename = locationPath.slice(0, indexOf + 7);
+const baseUrl = document.querySelector("base")?.href ?? "http://localhost:9091/";
```

Comment:
If there is a better way to tackle this, please feel free. cc @pierrejeambrun

Reply:
Why did we remove `const locationPath = globalThis.window.location.pathname;`? For tests?